Checking the Genuineness of your Skycoin Hardware Wallet.md

```diff
@@ -0,0 +1,64 @@
+# Checking the Genuineness of your SkyWallet
+
+## Checking your Skywallet Shipment Package for Tampering
+
+Once you receive the Skywallet shipment, please check for any physical tampering or damage of any sorts to the package.
+
+Tampering or damage can be anything from a small tear on the package to a package that is torn open.
+
+**Please do not accept a shipment which seems to be tampered in any such ways.**
+
+### An Untampered Shipment:
+
+<A Picture of an untampered shipment before opening>
+
+### A Tampered Shipment:
+
+<Optional picture of a tampered shipment which is not absolutely torn open but a small tear or hole so that the user will understand even such a small tamper can also spoil the device, it can also be a general picture, rather than a skywallet shipment>
+
+Upon opening the package, please ensure that the holographic sticker with the Skycoin logo on the box is not broken.
+<A Picture of the unbroken holographic seal on the box>
+
+## Checking the Firmware
+
+Skywallet is built to provide maximum security to the user.
+
+However, it is always important to check whether your Skywallet is not somehow compromised before adding your coins to the wallet.
+
+## Step 1 - Connecting Your Skywallet
+
+As the first step, you have to connect the device to your USB port.
+
+<Optional photo Showing the connection of the device to USB port>
+
+## Step 2 - Checking the message on the Wallet's Display
+
+### Scenario 1
+
+The display screen equipped on your Skywallet displays one of the following three messages when connected via USB port, marking the authenticity of your Wallet.
+
+If the Skywallet is genuine, then the screen will display the following message:
+
+**NEED SEED!**
+
+Along with the Skycoin Logo.
+
+<A photo of the Hardware wallet displaying NEED SEED message>
+
+### Scenario 2:
+
+A Previously used Skywallet will have a Seed set up beforehand.
+
+Hence the message **Need Seed!** will not be displayed and this denotes that your Skywallet is unsafe to use as someone else can access your wallet using that previously set up Seed.
+
+<A picture of the Hardware wallet which doesnot display the message NEED SEED>
+
+### Scenario 3:
+
+The third and final message that might be displayed is a warning:
+
+**Unofficial Firmware Detected**
+
+denoting that the firmware used by the Wallet is not an approved version of Skycoin Software, this is extremely dangerous as the software version on your Wallet might be corrupted and using the said wallet is ***unsafe***.
```

Comment:
Change to: `denoting that the firmware is not approved by Skycoin. Do **not** use a Skywallet displaying that warning as ....`